src/demo/App.vue

```diff
@@ -272,6 +276,12 @@ const pointsAxesLabelGridZoomAndLogScale = makeRandomPoints(pointPropsBasic);
 pointsAxesLabelGridZoomAndLogScale.forEach(p => p.x -= 0.5);
 const pointsPointsAxesAndZoom = makeRandomPoints(pointPropsBasic);
 const curvesTooltips = makeRandomCurves(propsBasic);
+const curvesArea = makeRandomCurves(propsBasic);
```

Comment:
I think it would be nice to have fewer lines on this demo chart -- it's ok to have loads of '1D' traces, but with this many it's hard to see the '2D' area layers clearly.
```suggestion
const curvesArea = makeRandomCurves({ ...propsBasic, nL: 5 });
```